pkg/settings/features.go

```diff
@@ -16,12 +17,15 @@ type Features struct {
 	RetainPrecopyImporterPods bool
 	// Whether to use changeID-based incremental backup workflow (with a version of CDI that supports it)
 	VsphereIncrementalBackup bool
+	// Wherther to use Volume Populators or use the default CDI workflow
+	VolumePopulator bool
 }
 
 // Load settings.
 func (r *Features) Load() (err error) {
 	r.OvirtWarmMigration = getEnvBool(FeatureOvirtWarmMigration, false)
 	r.RetainPrecopyImporterPods = getEnvBool(FeatureRetainPrecopyImporterPods, true)
 	r.VsphereIncrementalBackup = getEnvBool(FeatureVsphereIncrementalBackup, false)
+	r.VolumePopulator = getEnvBool(FeatureVolumePopulator, true)
```

Comment:
I think it doesn't worth it to add volume-populator as a 'Feature' now - for OpenStack we have no other choice and for oVirt we don't want to switch back to CDI in cold-migrations because we've already added stuff that relies on using the volume populator - so there's no sense in disabling it at this point

Reply:
Removed the new feature